test/python/transpiler/test_unitary_synthesis.py

```diff
@@ -673,38 +569,6 @@ def test_coupling_map_transpile_with_backendv2(self, opt_level, bidirectional):
                 (0, 1), (circ_01_index[instr.qubits[0]], circ_01_index[instr.qubits[1]])
             )
 
-    @data(1, 2, 3)
-    def test_coupling_map_unequal_durations(self, opt):
```

Comment:
I removed this test because we originally had it as the backend V1 version of `test_coupling_unequal_duration_with_backendv2`, it was then migrated to backend v2 and became essentially a duplicate of the other one.